docs/pages/push-notifications/receiving-notifications.mdx

```diff
@@ -34,6 +36,180 @@ useEffect(() => {
 }, []);
 ```
 
+<Collapsible summary="Android Notification Object Example from addNotificationReceivedListener">
```

Comment:
```suggestion
<Collapsible summary={<>Android notification object example from <CODE>addNotificationReceivedListener</CODE></>}>
```

Sentence case is preferred over title case for section headings as per our writing style guide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#headings